Userland/Libraries/LibLocale/DateTimeFormat.h

```diff
@@ -9,7 +9,6 @@
 #include <AK/DeprecatedString.h>
 #include <AK/Optional.h>
 #include <AK/StringView.h>
-#include <AK/Time.h>
```

Comment:
Same here, the full definition is needed.
(I also wouldn't call `Locale::format_time_zone` a "convenience helper" but :man_shrugging:)